R/lists_statuses.R

```diff
@@ -82,5 +50,15 @@ lists_statuses_ <- function(list_id = NULL,
     tweet_mode = "extended"
   )
 
-  TWIT_get(token, "lists/statuses", params)
+  results <- TWIT_paginate_max_id(token, "lists/statuses", params,
+    get_max_id = function(x) x$id_str,
+    page_size = 200,
```

Comment:
On the documentation we said something else: 

> n optional Specifies the number of results to retrieve per "page.".

Might be better to change the documentation to the max, number of tweets to get. 
Also about documentation, this function does not have any examples.